extensions/src/ACRE2Arma/common/easylogging++.h

```diff
@@ -15,7 +15,7 @@
 //
 #ifndef EASYLOGGINGPP_H
 #define EASYLOGGINGPP_H
-// Compilers and C++0x/C++11 Evaluation
+// Compilers and C++0x/c++17 Evaluation
```

Comment:
did you actually update easylogging??? Doesn't look like it, there are no changes besides string replacements?